codalab/lib/bundle_cli.py

```diff
@@ -600,20 +600,40 @@ def resolve_key_targets(self, client, worksheet_uuid, target_specs):
         Helper: target_specs is a list of strings which are [<key>]:<target>
         Returns: [(key, worker.download_util.BundleTarget), ...]
         """
-        keys = set()
+
+        def is_ancestor(path1, path2):
+            # Checks if a path is an ancestor of the other path (e.g. foo and foo/bar)
+            i = 0
```

Comment:
I think we simplify the implementation with something like `path2.startswith(path1 + '/')`

Reply:
Your suggestion was much simpler.